config-api/src/main/java/org/keycloak/config/TransactionOptions.java

```diff
@@ -5,17 +5,26 @@
 
 public class TransactionOptions {
 
-    public static final Option<Boolean> TRANSACTION_XA_ENABLED = new OptionBuilder<>("transaction-xa-enabled", Boolean.class)
+    public static final Option<Boolean> DB_XA_ENABLED = new OptionBuilder<>("db-xa-enabled", Boolean.class)
             .category(OptionCategory.TRANSACTION)
-            .description("Manually override the transaction type. Transaction type XA and the appropriate driver is used by default.")
+            .description("Set if the XA transaction type is used. Transaction type XA and the appropriate database driver is used by default.")
             .buildTime(true)
             .defaultValue(Boolean.TRUE)
             .expectedValues(Boolean.TRUE, Boolean.FALSE)
             .build();
 
+    public static final Option<Boolean> TRANSACTION_JTA_ENABLED = new OptionBuilder<>("transaction-jta-enabled", Boolean.class)
+            .category(OptionCategory.TRANSACTION)
+            .description("Set if JTA is used. Transaction type XA and the appropriate driver is used by default.")
```

Comment:
Set if distributed transactions are supported. If set to false, transactions are managed by the server and can not be joined if multiple data sources are used. By default, distributed transactions are enabled and only XA data sources can be used.